app/scripts/modules/openstack/loadBalancer/configure/wizard/listeners.html

```diff
@@ -18,16 +17,14 @@
         <tr ng-repeat="listener in loadBalancer.listeners">
           <td><select class="form-control input-sm" ng-model="listener.externalProtocol"
```

Comment:
OK. Not a big deal... and not worth putting in a translation in the UI code... just thought that having "protocol" instead of "externalProtocol" would be less confusing and therefore more maintainable.